cmd/compute/instance/instance_list.go

```diff
@@ -11,24 +11,51 @@ import (
 	exocmd "github.com/exoscale/cli/cmd"
 	"github.com/exoscale/cli/pkg/globalstate"
 	"github.com/exoscale/cli/pkg/output"
+	"github.com/exoscale/cli/table"
 	"github.com/exoscale/cli/utils"
 	v3 "github.com/exoscale/egoscale/v3"
 )
 
 type instanceListItemOutput struct {
-	ID        string `json:"id"`
-	Name      string `json:"name"`
-	Zone      string `json:"zone"`
-	Type      string `json:"type"`
-	IPAddress string `json:"ip_address"`
-	State     string `json:"state"`
+	ID          string `json:"id"`
+	Name        string `json:"name"`
+	Zone        string `json:"zone"`
+	Type        string `json:"type"`
+	IPAddress   string `json:"ip_address"`
+	IPv6Address string `json:"ipv6_address"`
+	State       string `json:"state"`
 }
 
 type instanceListOutput []instanceListItemOutput
 
-func (o *instanceListOutput) ToJSON()  { output.JSON(o) }
-func (o *instanceListOutput) ToText()  { output.Text(o) }
-func (o *instanceListOutput) ToTable() { output.Table(o) }
+func (o *instanceListOutput) ToJSON() { output.JSON(o) }
+func (o *instanceListOutput) ToText() { output.Text(o) }
+func (o *instanceListOutput) ToTable() {
+	t := table.NewTable(os.Stdout)
+	defer t.Render()
+
+	t.SetHeader([]string{
+		"ID",
+		"NAME",
+		"ZONE",
+		"TYPE",
+		"IP ADDRESS",
+		"IPV6 ADDRESS",
```

Comment:
I think we can merge these 2 columns into "IP ADDRESS" to de-clutter interface (table output only).
Then just put IPv6 in the parentheses, eg, `0.0.0.0 (0a00:c00:c00:0f0e:000:0cff:fe00:00)` or just `0.0.0.0` when no IPv6 exists. IPv4 and IPv6 formats are different so there will be no confusion.
cc @pierre-emmanuelJ @sauterp

Reply:
the thing is that with the separate column they are aligned more accurately at least